src/Apps/Artist/Routes/AuctionResults/Components/EmptyArtistAuctionResults.tsx

```diff
@@ -0,0 +1,32 @@
+import { Flex, Text, Button, GridColumns, Column } from "@artsy/palette"
+import { RouterLink } from "System/Router/RouterLink"
+
+export const EmptyArtistAuctionResults: React.FC = () => {
+  return (
+    <Flex>
+      <GridColumns width="100%" gridRowGap={2}>
+        <Column span={6} start={4}>
+          <Text variant="md" textAlign="center">
+            There are no auction results for this artist at this time.
+          </Text>
+        </Column>
+        <Column span={6} start={4}>
+          <Text variant="md" color="black60" textAlign="center">
+            The page will be updated once auction results are added. In the
+            meantime, check out the millions of auction results and art market
+            data.
+          </Text>
+        </Column>
+        <Column span={12} alignItems="center">
+          <Flex width="100%" justifyContent="center">
```

Comment:
You can just `display=flex` the column, though I think you might just be looking for `mx="auto"` on the button.

Reply:
I don't quite understand this. Does the `display=flex` in the column mean, I can get rid of the wrapping `Flex`? And what would the `mx=auto` do on the button? I will try it out a bit but I don't really know what I am doing.